src/db/migrations/0010_flashy_wendell_vaughn.sql

```
@@ -0,0 +1,99 @@
+SET
+	"sandbox_id" = NULLIF("metadata"->>'sandboxId', ''),
+	"metadata" = "metadata" - 'sandboxId'
+WHERE "metadata" ? 'sandboxId';--> statement-breakpoint
+UPDATE "sandbox_jobs"
+SET
+	"provisioning_claimed_at" = COALESCE("started_at", "created_at"),
+	"provisioning_expires_at" = COALESCE("started_at", "created_at") + INTERVAL '31 minutes'
+WHERE
+	"sandbox_id" IS NULL
+	AND "status" IN ('pending', 'running', 'canceling');--> statement-breakpoint
+CREATE INDEX "sandbox_jobs_run_id_sandbox_id_idx" ON "sandbox_jobs" USING btree ("run_id","sandbox_id");--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)